packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -278,7 +394,7 @@ const AccountTransferForm = ({
         recentTransactionOnMount();
     }, [recentTransactionOnMount]);
 
-    const validateAmount = amount => {
+    const validateAmount = (amount: string | number) => {
```

Comment:
Do we really pass both number and string to this method?
Shouldn't it be only string?

Reply:
Fixed.